src/components/Nav.jsx

```diff
@@ -5,17 +5,71 @@ import brigadeLogo from '../images/cfc_logo_2021.svg'
 import slackLogo from '../images/slack.svg'
 import githubLogo from '../images/github.svg'
 import meetup from '../images/meetup.svg'
+import styled from 'styled-components'
+
+const StyledNav = styled.nav`
```

Comment:
Let's nest _all_ other styles within this styled nav. I do this for a few reasons:

1. The markup looks cleaner (you can immediately tell what element type everything is).
2. It's less typing. 
3. Most of the CSS looks like regular SASS (which is an advantage to anyone who's used SASS before and is used to it).

Let me know if you have any questions about this. This definitely is not the wrong way to do it. I've seen many great devs use this approach. I just prefer to nest styles in one parent styled component (though there are definitely exceptions).